plugins/pre-commit.sh

```diff
@@ -29,38 +29,34 @@ fi
 ########## PIP VERSION #############
 # Verify that yapf is installed; if not, warn and exit.
 if ! command -v yapf >/dev/null; then
-  echo 'yapf not on path; can not format. Please install yapf:'
-  echo '    pip install yapf'
+  printf 'yapf not on path; can not format. Please install yapf:\n' \
```

Comment:
this unfortunately prints a message that lacks a trailing newline which can be bad for a CLI tool

you should just leave this comment block alone as `echo` works fine here